src/index.ts

```diff
@@ -36,6 +36,14 @@ const namesToCodes = {
   'InsufficientBalanceError': 'F08'
 }
 
+const toBrowserSafeURL = (btpUrl: string): string => {
+  if (!btpUrl.startsWith('btp+')) {
+    throw new Error('server must start with "btp+". server=' + this._server)
```

Comment:
^this

Reply:
Flip nice catch

Reply:
Any reason not to just return an URL as per toBrowserSafe*URL* ?

Reply:
Returning the URL now and fixed the other issue at the bottom